attest/api/src/convert/enclave_report_data_contents.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2023 The MobileCoin Foundation
+
+//! Convert to/from attest::EnclaveReportDataContents
+
+use crate::{attest, convert::encode_to_protobuf_vec, ConversionError};
+use mc_attest_verifier_types::{prost, EnclaveReportDataContents};
+use mc_util_serial::Message;
+use protobuf::Message as ProtoMessage;
+
+impl From<&EnclaveReportDataContents> for attest::EnclaveReportDataContents {
+    fn from(src: &EnclaveReportDataContents) -> Self {
+        let prost = prost::EnclaveReportDataContents::from(src);
```

Comment:
This is pretty similar to the `collateral.rs` version, with the difference being the `prost::TYPENAME` and the use of `from` instead of `try_from`.
I'm not sure it's worth investing in macroizing or genericizing these at this time.